### DISCUSSION TITLE
My limit rate is maxing out way below 15,000 requests

### DISCUSSION THREAD

**Discussion by kris-sitback** [soundcloud/api#225] [Q&A]
Hi there,
I'm testing on my local and finding the rate limit is maxing out at around maybe 100 requests or so. I read in the api docs that the limit is 15,000 requests per 24 hours. Any insight why I'm getting a 429 error when I'm way below the 15,000 request mark?
Cheers

**Comment by mgoodfellow** [comment]
I'm not sure this repo is active anymore - but I can try to give you some info.

The rate limit you mention of 15,000 is regarding stream of tracks in 24 hours and that is due to licensing: https://developers.soundcloud.com/docs/api/rate-limits#play-requests

There are other various rate limits depending on the endpoints. Without you providing some context or extra information about the exact endpoints and methods you are calling and whether you are referring to server or user context, I cannot offer much more insight.

Other rate limits:
* Global edge rate limits enforced against IP against all SC infrastructure - I believe 100 calls rolling window over a 10s https://github.com/soundcloud/api/issues/189#issuecomment-1063957741
* Client credentials token grant by client ID - 50 calls in 24hr https://github.com/soundcloud/api/releases/tag/2021-09-07
* Client credentials token grant by IP - 30 calls per hour https://github.com/soundcloud/api/issues/223
* User action rate limits (user context) - for example the user following too many people etc. No idea what is limited, or levels, but generally user interactions are limited for obvious reasons

**Comment by dpreussler** [comment]
closing as of age, feel free to reopen if needed
